packages/core/src/lib/core/mutations/index.ts

```diff
@@ -304,6 +306,54 @@ export async function deleteMany(
   })
 }
 
+export async function actionOne(
+  where: UniqueInputFilter,
+  list: InitialisedList,
+  context: KeystoneContext,
+  action: InitialisedAction
+) {
+  const operationAccess = await action.access({
+    context,
```

Comment:
Missing `session: context.session`, added in https://github.com/keystonejs/keystone/commit/51e767059c78ba1c0b1114c4e24aa8d6a2aa6e69